js-sdk/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "gitcoin-sdk",
+  "version": "1.0.2",
+  "description": "Gitcoin SDK.",
+  "scripts": {
+    "dev": "webpack --config webpack.config.js --mode development --progress --colors",
+    "build": "webpack --config webpack.config.js --mode production --progress --colors",
+    "prepublish": "npm run build",
+    "eslint": "./node_modules/eslint/bin/eslint.js --quiet --config .eslintrc ./src/*.js",
+    "eslint:fix": "./node_modules/eslint/bin/eslint.js --quiet --fix --config .eslintrc ./src/*.js",
+    "stylelint": "stylelint ./src/*.css",
+    "stylelint:fix": "stylelint --fix ./src/*.css",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": ["Gitcoin", "SDK"],
+  "author": "Matthew Lilley",
```

Comment:
@owocki  Do we want this updated ?

Reply:
where is the info displayed? on npm somewhere?

Reply:
https://www.npmjs.com/package/gitcoin-sdk

You can edit this and republish if you wish. I'll need somebody to take control of the NPM package on npmjs. 

Do you have an npmjs org account for Gitcoin? 

If not, go ahead and create one https://www.npmjs.com/org/create, once done I'll transfer the package over to it.

Reply:
I think I need to transfer it to a user which has access to the org. Can you confirm your NPM username?

Reply:
@matthewlilley `mbeacom` 😂

Reply:
@mbeacom I had to make sure 😂 Done!